src/index.ts

```diff
@@ -1,10 +1,3 @@
-import * as core from '@actions/core';
```

Comment:
Is this a best-practice you've seen elsewhere? Abstracting out the main thread from the index into another file?

Reply:
Yeah, I referenced the official Typescript actions template (https://github.com/actions/typescript-action/blob/main/src/index.ts) and the labeler action (https://github.com/actions/labeler/blob/main/src/main.ts)

Reply:
I'm ok with that. It's another level of abstraction but makes testing more intuitive 👍